src/test/java/com/atlassian/bitbucket/jenkins/internal/client/BitbucketClientFactoryImplTest.java

```diff
@@ -40,103 +46,89 @@
 public class BitbucketClientFactoryImplTest {
 
     private static final String BASE_URL = "http://localhost:7990/bitbucket";
+
     private BitbucketClientFactoryImpl anonymousClientFactory;
-    @Mock
-    private ResponseBody mockBody;
-    @Mock
-    private Call mockCall;
-    @Mock
-    private OkHttpClient mockClient;
+    private MockRemoteHttpServer mockRemoteHttpServer = new MockRemoteHttpServer();
```

Comment:
nitpick: this can be final, and I know its not your code, but the `objectMapper` below can be static final